ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -17,13 +17,14 @@ http {
     access_log /logs/nginx_access.log combined-upstream;
 
     server {
+        listen 80;
```

Comment:
we could perhaps enable this via an ansible flag (forceEnableHttpPort)? the point is that we're not testing this so it should be clear this isn't standard.